### PR TITLE
add "runtime" variable to select between runc and cc-oci-runtime

### PR DESCRIPTION
**Pull request by albertomurillo**
Signed-off-by: Alberto Murillo Silva <alberto.murillo.silva@intel.com>

### CODE REVIEW

**Comment by albertomurillo** [comment]
This only works for Ubuntu and Fedora for now

**Comment by runcom** [comment]
:+1: @mrunalp ptal

File: cri-o.yml [hunk 1]
```diff
@@ -169,6 +171,22 @@
         regexp: 'overlay2\.override_kernel_check=1'
         state: present
       when: (ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS')
+    - block:
```
Comment (mrunalp) [inline cri-o.yml]: Does this allow us to bring up a cluster that uses runc for all trused and untrusted workloads? If not, we should have a env variable/parameter to control that.

Comment (albertomurillo) [reply cri-o.yml]: This commit includes `group_vars/all` file which sets `runtime` variable to `runc` by default.

User can override that variable int he `group_vars/all` file, in the hosts file or in the cmdline

Comment (runcom) [reply cri-o.yml]: Mmm is there anyway to install a both to use the mixed workload for cri-o?

**Comment by albertomurillo** [comment]
@runcom Is there anything else you want me to add to this PR before merging?

**Pull request merged**